examples/pylintrc

```diff
@@ -1,5 +1,22 @@
 [MASTER]
```

Comment:
Would you mind putting this in a separate PR/commit? It's not really related to the bump but rather to a (quite large) regeneration of the file.

Reply:
I'm going to do that. It's related to the bump because of https://github.com/PyCQA/pylint/blob/main/tbump.toml#L36, I think it will be waaaay shorter once the argparse migration is behind us.

Reply:
If we're investing time in this:
what about adding ``--generate-toml-config`` as an example as well?